examples/source/1.components/amp-sidebar.html

```diff
@@ -42,102 +44,178 @@
+      -->
+      <div>
+        <amp-sidebar id="sidebar-left" class="sample-sidebar" layout="nodisplay" side="left">
+          <h3>Sidebar</h3>
+          <button on="tap:sidebar-left.close">Close sidebar</button>
+          <nav toolbar="(min-width: 784px)" toolbar-target="target-element-left">
+            <ul>
+              <li>Nav item 1</li>
+              <li>Nav item 2</li>
+            </ul>
+          </nav>
           <ul>
-            <li>Nav item 1</li>
-            <li>Nav item 2</li>
+            <li>Nav item 3</li>
+            <li>Nav item 4</li>
           </ul>
-        </nav>
-        <ul>
-          <li>Nav item 3</li>
-          <li>Nav item 4</li>
-        </ul>
-      </amp-sidebar>
-      <button on="tap:sidebar-right.toggle">Toggle sidebar</button>
-      <div id="target-element-right">
+        </amp-sidebar>
+        <button on="tap:sidebar-left.toggle">Toggle sidebar</button>
+        <div id="target-element-left">
+        </div>
       </div>
-    </div>
 
-    </body>
+      <h2 class="previewOnly">Styled toolbar</h2>
+      <!-- ## Styled toolbar -->
+      <!--
+        The `toolbar` element within the `<amp-sidebar>` element, will have classes applied to the element depending if the `toolbar-target` element is shown or hidden. This is useful for applying different styles on the `toolbar` element and the `toolbar-target` element. The classes are `amp-sidebar-toolbar-target-shown`, and `amp-sidebar-toolbar-target-hidden`. The class `amp-sidebar-toolbar-target-shown` is applied to the `toolbar` element when the `toolbar-target` element is shown. The class `amp-sidebar-toolbar-target-hidden` is applied to the `toolbar` element when the `toolbar-target` element is hidden.
+
+        ```css
+        .amp-sidebar-toolbar-target-shown {
+          display: none;
+        }
+        ```
+        -->
+        <div>
+          <amp-sidebar id="sidebar-right" class="sample-sidebar" layout="nodisplay" side="right">
+            <h3>Sidebar</h3>
+            <button on="tap:sidebar-right.close">Close sidebar</button>
+            <nav toolbar="(min-width: 784px)" toolbar-target="target-element-right">
+              <ul>
+                <li>Nav item 1</li>
+                <li>Nav item 2</li>
+              </ul>
+            </nav>
+            <ul>
+              <li>Nav item 3</li>
+              <li>Nav item 4</li>
+            </ul>
+          </amp-sidebar>
+          <button on="tap:sidebar-right.toggle">Toggle sidebar</button>
+          <div id="target-element-right">
+          </div>
+        </div>
+    </article>
+  </main>
+  <!-- ## Desktop Sidebar -->
+  <!--
+    The amp-sidebar can be used as a permanent navigation bar on the main page.
+    The toolbar target is defined as a left hand navigation element using css: flex.
+    (See the `aside` element under `main`.)
```

Comment:
Instead of referring the reader elsewhere on the page, let's provide a code snippet of roughly what's going on. The important information is that the sidebar behavior is available as usual below a given width threshold and permanently affixed above it. This is a skeleton of what you could describe in the comments here:
```html
<main>
   <aside> ... target for the sidebar above the min-width threshold </aside>
</main>
<amp-sidebar ... > ... </amp-sidebar>
```

Reply:
See my suggestions above. Integrate header and main here (ideally via three consecutive code snippets).

Reply:
Thanks for the feedback Caroline, Sebastian.  I have updated this section with three code snippets, bringing in the aside and CSS from previous sections.  This will make the flow of the page more logical and keeps the user from having to scroll around the page.  Thanks!